README.rst

```diff
@@ -128,25 +142,32 @@ have a tiling that corresponds to non-empty permutation avoiding
        012: (0, 0), (2, 0), (2, 0)
        Requirement 0:
        0: (1, 1)
-       >>> til.dimensions
+       >>> tiling.dimensions
```

Comment:
Maybe stop this code cell before calling the dimensions. And add text before continuing another code cell, e.g., "There are several properties of tilings you can call... e.g.,"

Reply:
Added